examples/jsm/renderers/CSS2DRenderer.js

```diff
@@ -212,4 +212,6 @@ class CSS2DRenderer {
 
 }
 
+CSS2DRenderer.DEFAULT_MATRIX_WORLD_AUTO_UPDATE = true;
```

Comment:
TBH, I don't think this flag is required since you can use the existing API to achieve the same thing. Just set `scene.matrixWorldAutoUpdate` to `false` when creating the scene and then call `scene.updateMatrixWorld()` in your animation loop before updating the renderers.

Reply:
Thanks for your reply, It's a great way to solve my problem.

I really didn't think of it this way, thanks to you again.  I will close the PR at once.